From 8acda2cea4688b8fca910ea8780fc29594ba6085 Mon Sep 17 00:00:00 2001
From: David Smith <dsmith@redhat.com>
Date: Mon, 11 Apr 2016 14:07:25 -0500
Subject: [PATCH] Fixed PR19940 by updating runtime/linux/access_process_vm.h.

* runtime/linux/access_process_vm.h: Changed page_cache_release() to
  put_page().
---
 runtime/linux/access_process_vm.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/runtime/linux/access_process_vm.h b/runtime/linux/access_process_vm.h
index fa11baf0276f..a2114c60d9c9 100644
--- a/runtime/linux/access_process_vm.h
+++ b/runtime/linux/access_process_vm.h
@@ -52,7 +52,7 @@ __access_process_vm_ (struct task_struct *tsk, unsigned long addr, void *buf,
 	  reader (vma, page, addr, buf, maddr + offset, bytes);
 	}
       kunmap (page);
-      page_cache_release (page);
+      put_page (page);
       len -= bytes;
       buf += bytes;
       addr += bytes;
-- 
2.8.1

